.github/workflows/tests.yml

```diff
@@ -80,7 +80,7 @@ jobs:
       - resolve-versions
       - static-checks
     runs-on: ${{ matrix.os }}
-    timeout-minutes: 10
+    timeout-minutes: 20
```

Comment:
Another GH workflow related change, it seems that `setup-go` is taking an extra 2 minutes to setup Go `1.22.x` on `windows-latest`, not leaving enough time for the rest of the job to run 🙂 : https://github.com/hashicorp/terraform-exec/actions/runs/8914407327/job/24482771576?pr=447